src/lektorium/repo/local/repo.py

```diff
@@ -116,12 +145,13 @@ def create_session(self, site_id, custodian=None):
         self.storage.create_session(site_id, session_id, session_dir)
         session_object = Session(
             session_id=session_id,
-            creation_time=datetime.datetime.now(),
-            view_url=None,
-            edit_url=self.server.serve_lektor(session_dir),
+            site_id=site_id,
+            creation_time=datetime.now(),
             custodian=custodian,
             custodian_email=custodian_email,
         )
+        session_object['edit_url'] = self.server.serve_lektor(session_dir, session_object)
+        session_object.pop('site_id', None)
```

Comment:
Why remove `site_id` from session object?

Reply:
I just fixed this part. Will commit in a minute.